Client/src/Pages/Monitors/Home/actionsMenu.jsx

```diff
@@ -44,6 +47,28 @@ const ActionsMenu = ({ monitor, isAdmin, updateCallback }) => {
     }
   };
 
+  const handlePause = async (event) => {
+    try {
+      const action = await dispatch(
+        pauseUptimeMonitor({ authToken, monitorId: monitor._id })
+      );
+      if (pauseUptimeMonitor.fulfilled.match(action)) {
+        updateCallback();
+        //window.location.reload();
+        if (action.payload?.isActive === false){
+          createToast({ body: "Monitor paused successfully." });
+        }else{
+          createToast({ body: "Monitor resumed successfully." });
+        }
+      } else {
+        throw new Error(action.error.message);
+      }
+    } catch (error) {
+      console.error("Error pausing monitor:", monitor._id, error);
```

Comment:
We have a utility Logger that we should use for logging:

```
  import { logger } from "../../../Utils/Logger";
  ...
  logger.error("Error pausing monitor:", monitor._id, error);
```

The logger was added mid-projecyt, so if you see `console.log` statements throughout the app we just haven't gotten around to replacing them yet.